### ISSUE TITLE
Msvcp120.dll missing

### ISSUE THREAD

**Issue by cwcriner** [Cockatrice/Cockatrice#1582]
so cockatrice won't launch because msvcp120.dll is missing,  visual C++ 2010 says I got a newer version, so I try 2013 and 2015, same result.  I'm running windows 7. what's my next step?

**Comment by ZeldaZach** [comment]
https://github.com/Cockatrice/Cockatrice/wiki/Troubleshooting#failed-start-because-msvcp120dll-was-not-found

**Comment by Aitrusx** [comment]
I can hopefully piggyback on this - I'm on windows 10 with the same mscvcp120.dll missing. When I download off the above link it won't install anything because it says windows has a more recent version already. 

My windows is fully updated. I have deleted all my cockatrice files including decks and pictures, restarted and shut down/on multiple times, every install of cockatrice (from here: http://www.reddit.com/r/Cockatrice/comments/3m1cwr/the_expeditions_have_arrived/cvb79ao) results in the same thing - program won't run because it says that .dll is missing :(

**Comment by Daenyth** [comment]
@woogerboy21 Do we need a new build just for windows 10 maybe?

**Comment by woogerboy21** [comment]
@Daenyth as in linking to some type of different runt time file set?  I would hope not as were getting to the point in windows we would now be creating 3 sets of installers :(

Others are running the app fine in windows 10 so there's bound to be a solution.

**Comment by woogerboy21** [comment]
What is odd is I read everyone saying they are installing the installer from the URL:
http://www.microsoft.com/en-us/download/details.aspx?id=40784

Are you sure you guys are downloading the 32bit installer (not the 64bit) and running it?

**Comment by DreadnoughtMTG** [comment]
I have this issue also. It started after downloading the newest client. I tried finding the correct vcredist_x86 file but nothing seemed to change. There seems to be an issue somewhere with windows 10 compatibility. I performed a software update to no avail also. I was able to revert to my older client which will open but will not login to the server any longer.

**Comment by DreadnoughtMTG** [comment]
Alright, after trying this out again, it's now working. I went ahead and used the link woogerboy21 posted for my vcredist_86 download and this time something clicked. Program opens and connects with no issues

**Comment by woogerboy21** [comment]
I have a hunch people are downloading the x64 version.  My guess is that its like IE telling them what version they should download based on there detected OS type.  But just a guess.

**Comment by Aitrusx** [comment]
So either the troubleshooting FAQ was updated or my chrome/internet was just being crazy. Previously the link provided for the C++ download from microsoft would provide me one preset option, I would download it and windows 10 refused to install it.

This morning I tried https://www.microsoft.com/en-us/download/details.aspx?id=40784 and was presented with 3 different .exe files. I chose the x86 one, and it downloaded and actually installed and I was then able to open cockatrice with no .dll error. Did not get a chance to try oracle or connect to the server but I would assume its all functioning correctly now - was just a matter of getting the right link to download the C++ from
